tests/sympc/tensor/mpc_tensor_test.py

```diff
@@ -250,6 +250,27 @@ def test_ops_public_mpc(get_clients, nr_clients, op_str) -> None:
     assert np.allclose(result, expected_result, atol=10e-4)
 
 
+falcon = Protocol.registered_protocols["Falcon"]()
+
+
+@pytest.mark.parametrize("op_str", ["add", "sub"])
+def test_ops_public_mpc_falcon(get_clients, op_str) -> None:
+    clients = get_clients(3)
```

Comment:
Have tests with different parties.

Reply:
I am waiting for the Default Protocol PR(https://github.com/OpenMined/SyMPC/pull/231).Once it is merged I will include it.